frontend/src/pages/article/article.tsx

```diff
@@ -0,0 +1,11 @@
+import { type FC } from 'react';
+import { useParams } from 'react-router-dom';
+
+import { Author } from './components/author/author.js';
```

Comment:
@andanf-e  Shouldn't the `Author` component be imported from `~/libs/components/components.js`?

Reply:
I put it there because I suggest it is not a common component, but just a part of the article page

Reply:
Yep, @kovAnya is right - it's not a common component